src/server.h

```diff
@@ -2007,8 +2007,8 @@ struct valkeyServer {
                                                 * use dual connection for full syncs. */
     int wait_before_rdb_client_free;           /* Grace period in seconds for replica main connection
                                                 * to establish psync. */
-    int debug_sleep_after_fork_ms;             /* Debug param that force the main connection to
-                                                * sleep for N ms after fork() in repl. */
+    int debug_sleep_after_fork_us;             /* Debug param that force the main connection to
+                                                * sleep for N וs after fork() in repl. */
```

Comment:
What's that unicode symbol? Wrong encoding of "µ"?

Maybe better use ascii and spell out "microseconds"?